spec/classroom_spec.rb

```diff
@@ -0,0 +1,10 @@
+require './classroom'
+require './student'
+
+describe Classroom do
+  # write the test here
+  it 'should have a label' do
+    classroom = Classroom.new('Math')
+    classroom.label.should == 'Math'
+  end
+end
```

Comment:
-  [ ] [REQUIRED] Hello team, great job so far but it seems some methods have not been tested.
Let's test the `add_student` method for this class as stated in the project requirements.